system-test/storage.ts

```diff
@@ -3866,6 +3866,64 @@ describe('storage', function () {
     });
   });
 
+  describe('universeDomainTests', () => {
+    const TEST_UNIVERSE_DOMAIN = process.env.TEST_UNIVERSE_DOMAIN;
```

Comment:
@JesseLovelace brought up a good point, we should probably assert that these credentials and environment variables are not undefined / null.

Reply:
Thanks for bringing that up, I've implemented validations to ensure that the credentials and environment variables are not undefined or null.  Added kokoro changes again. 

Kindly review and rerun kokoro.